riscv/mmu.cc

```diff
@@ -76,7 +76,7 @@ reg_t mmu_t::translate(reg_t addr, reg_t len, access_type type, uint32_t xlate_f
 
 tlb_entry_t mmu_t::fetch_slow_path(reg_t vaddr)
 {
-  check_triggers(triggers::OPERATION_EXECUTE, vaddr, false);
+  check_triggers(triggers::OPERATION_EXECUTE, vaddr, std::nullopt);
```

Comment:
Want to make `std::nullopt` the default value for `data`, so we can just leave off the last parameter here?

Reply:
sgtm